system/system_monitor/include/system_monitor/net_monitor/net_monitor.hpp

```diff
@@ -101,9 +118,17 @@ class NetMonitor : public rclcpp::Node
   std::vector<std::string> device_params_;  //!< @brief list of devices
   NL80211 nl80211_;                         // !< @brief 802.11 netlink-based interface
 
-  std::string monitor_program_;  //!< @brief nethogs monitor program name
-  bool nethogs_all_;             //!< @brief nethogs result all mode
-  int traffic_reader_port_;      //!< @brief port number to connect to traffic_reader
+  std::deque<unsigned int>
+    reasm_fails_queue_;  //!< @brief queue that holds count of IP packet reassembles failed
+  uint64_t
+    last_reasm_fails_;  //!< @brief IP packet reassembles failed at the time of the last monitoring
+
+  std::string monitor_program_;              //!< @brief nethogs monitor program name
+  bool nethogs_all_;                         //!< @brief nethogs result all mode
+  int traffic_reader_port_;                  //!< @brief port number to connect to traffic_reader
+  unsigned int reasm_fails_check_duration_;  //!< @brief IP packet reassembles failed check duration
```

Comment:
Please consider to change following the given above.

Reply:
@ito-san 
I changed the word as suggested.
24a196af79dda9c44c2781252396b25fd5df32f2